samples/NativeIISSample/NativeIISSample.csproj

```diff
@@ -5,7 +5,18 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Server.IISIntegration\Microsoft.AspNetCore.Server.IISIntegration.csproj" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
+    <!-- x86 -->
+    <None Include="..\..\src\RequestHandler\bin\$(Configuration)\Win32\aspnetcorerh.dll" CopyToOutputDirectory="PreserveNewest" Visible="false" Link="x86\aspnetcorerh.dll" />
```

Comment:
Does the global shim pickup `<app dir>/{x86,x64}/aspnetcorerh.dll`?

Reply:
It will look in the directory that aspnetcore.dll is loaded from for the requesthandler.

Reply:
👍 cool. thanks for clarifying